rust_src/src/charset.rs

```diff
@@ -1,11 +1,15 @@
 //! Basic character set support.
 
 use remacs_macros::lisp_fn;
+use std::ptr;
```

Comment:
Forgive the nitpicking please.

The import order is:
 - std
 - external crates
 - internal crates.

Reply:
No worries! I'll fix this. It looks like my editor reordered the lines because I placed them too close together.

Reply:
not your editor, that was rustfmt.